src/main/java/io/getstream/chat/java/models/App.java

```diff
@@ -794,6 +798,14 @@ public AppGetRateLimitsRequest endpoint(@NotNull String endpoint) {
     @JsonProperty("skip_devices")
     private Boolean skipDevices;
 
+    @Nullable
+    @JsonProperty("push_provider_name")
+    private String pushProviderName;
+
+    @Nullable
+    @JsonProperty("push_provider_type")
+    private PushProvider.PushProviderType pushProviderType;
```

Comment:
seems we're duplicating this between device and app where device one is missing huawei and xiaomi

Reply:
unified ✅